documentation/modules/security/proc-recovering-expired-ca-certs.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-recovering-expired-ca-certs-{context}']
+
+= Manually recovering from expired Cluster Operator-managed CA certificates
+
+[role="_abstract"]
+Cluster and clients CA certificates are automatically renewed by the Cluster Operator at the start of their renewal periods. 
+However, under certain circumstances, the renewal process may fail due to operational issues or disruptions. 
+In such an event, it is important to recover your deployment from using expired certificates to ensure secure communication. 
```

Comment:
Maybe we can word it something like that?

```suggestion
Once the certificates are expired, the different Kafka cluster components cannot communicate with each other and the Strimzi operator cannot renew the CA on its own without any further help.
In such an event, you should follow this recovery procedure. 
```

It explains that once it expires, it is simply broken and the operator cannot recover itself. _Ensuring secure communication_ is also a bit misleading as it is broken - not insecure.

Reply:
Changed to:

> The Cluster Operator automatically renews the cluster and clients CA certificates when their renewal periods begin. 
> Nevertheless, unexpected operational problems or disruptions may prevent the renewal process, such as prolonged downtime of the Cluster Operator or unavailability of the Kafka cluster. 
> If CA certificates expire, Kafka cluster components cannot communicate with each other and the Cluster Operator cannot renew the CA certificates without manual intervention.